### PR TITLE
Add HeyDL Universal Provider

### PR DESCRIPTION
**Pull request by Teekzilla**
Leaving this here as it currently works for some cases.

1. Movies should work as intended.
2. Recent TV shows which point to direct links are retrieved.

Similar to FardaDL, older shows may point to just a directory. Thus this will need to be updated to handle those cases. I tested `Westworld S01`, `Aquaman`, and `Spider-Man: Into the Spider-Verse`. I'll update as I make progress. Thinking to have this done tomorrow morning! If someone wants to test at the current state for any issues that would also be appreciated but no necessary.

### CODE REVIEW

**Comment by Teekzilla** [comment]
Updated to handle those folder links.

**Comment by kevincosmos** [comment]
Functional testing passed

File: src/scrapers/providers/index.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { queue } = require('../../utils/queue');
 
 module.exports.providers = {
     anime: [
-        new (require('./anime/AnimePahe'))(),
+        // new (require('./anime/AnimePahe'))(),
```
Comment (ddivad) [inline src/scrapers/providers/index.js]: Is there a reason this is commented out?

Comment (Teekzilla) [reply src/scrapers/providers/index.js]: Yes, because I missed it...

**Pull request merged**